client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.scss

```diff
@@ -0,0 +1,51 @@
+
+.stats-tier-upgrade-slider {
+	margin: 2px;
+	padding: 10px;
+
+	.jp-components-pricing-slider__thumb {
+		border-radius: 4px;
+		border-color: #008710;
+		background-color: #008710;
+		color: #fff;
+	}
+
+	.jp-components-pricing-slider__track {
+		height: 4px;
+	}
+
+	.jp-components-pricing-slider__control .mark {
```

Comment:
Could you split this into two selectors: `.jp-components-pricing-slider__control { .mark { } }`?